mysys/errors.cc

```diff
@@ -139,6 +139,99 @@ const char *globerrs[GLOBERRS] = {
     "Reset character out of range: %s.",
     "Unknown LDML tag: '%.*s'."};
 
+const char *globerrnames[GLOBERRS] = {
```

Comment:
It's better to define a single array of scructs with two string fields (one for mnemonic and another one for message) and define its content in one place rather than keeping two separate arrays (`globerrnames` and `globerrs`). This will help to avoid unsynchronized state problem when the list of errors needs to be extended.